packages/openactive-integration-tests/test/templates/README.md

```diff
@@ -0,0 +1,3 @@
+# Request Templates
+
+TODO
```

Comment:
Issue to create this: https://github.com/openactive/openactive-test-suite/issues/566